service/watcher/chartstatus/g8sclient.go

```diff
@@ -41,7 +41,7 @@ func (c *ChartStatusWatcher) waitForG8sClient(ctx context.Context) (versioned.In
 
 	o := func() error {
 		kubeConfigName := fmt.Sprintf("%s-kubeconfig", c.appNamespace)
-		restConfig, err = kubeConfig.NewRESTConfigForApp(ctx, kubeConfigName, c.appNamespace)
+		restConfig, err = kubeConfig.NewRESTConfigForApp(ctx, kubeConfigName, c.appNamespace, "values")
```

Comment:
We have kubeconfig secrets which have a key `kubeConfig`, so I think we need a fallback mechanism. (this is why we are failing in the integration-test)
1. check `kubeConfig` key first 
2. if not, check `values` key